lib/BloqadeQMC/src/ising/Rydberg.jl

```diff
@@ -69,86 +71,38 @@ function make_prob_vector(H::Type{<:AbstractRydberg}, V::UpperTriangular{T}, Ω:
     return ops, p, energy_shift
 end
 
+total_hx(H::Rydberg)::Float64 = sum(H.Ω) / 2
+haslongitudinalfield(H::AbstractRydberg) = !iszero(H.δ)
 
-###############################################################################
-
-# function BlockadeRydberg(dims::NTuple{N, Int}, J::Float64, hx::Float64, hz::Float64, pbc=true) where N
-# end
+function _make_vector(param::Real, Ns::Int64)
+    return param*ones(Ns)
+end
 
-function Rydberg(dims::NTuple{D, Int}, R_b, Ω, δ; pbc=true, trunc::Int=0, epsilon::Float64=0) where D
-    if D == 1
-        lat = Chain(dims[1], 1.0, pbc)
-    elseif D == 2
-        lat = Rectangle(dims[1], dims[2], 1.0, 1.0, pbc)
-    else
-        error("Unsupported number of dimensions. 1- and 2-dimensional lattices are supported.")
-    end
-    return Rydberg(lat, R_b, Ω, δ; trunc=trunc, epsilon=epsilon)
+function _make_vector(param::AbstractVector, Ns::Int64)
```

Comment:
```suggestion
function _make_vector(param::AbstractVector, Ns::Int)
```
see previous comment